src/core/biometric-auth/biometric-auth.web.ts

```diff
@@ -1,25 +0,0 @@
-import { AuthenticateConfig } from 'react-native-touch-id';
```

Comment:
Extension build is going to fail. You only need to remove `AuthenticateConfig` and `authenticate`